src/js/i18n/locales/ar.json

```diff
@@ -25,7 +25,7 @@
   "Menu.ThreeOneOneMobileListItem.callText": "Call 311",
   "Menu.ThreeOneOneMobileListItem.onlineText": "Submit an Online Request",
   "Menu.PrivacyPolicyListItem.text": "Read About Privacy",
-  "Menu.MobileFooter.text": "Alpha.austin.gov is a work in progress. For the full City of Austin website, visit",
+  "Menu.MobileFooter.text": "Alpha.austin.gov هو موقع إلكتروني جديد والعمل قيد التنفيذ. للحصول على كامل موقع مدينة أوستن، قم بزيارة {citySiteLink}. تعرف على المزيد حول الموقع الإلكتروني الجديد عبر {citySiteLink}.",
```

Comment:
should the last link be {projectSiteLink}? This text looks a lot like line 34 "Home.Secondarycontent.bodytext" but the last link is different.

Reply:
It's actually supposed to be a shortened version of "Home.Secondarycontent.bodytext". Good catch.